lms/djangoapps/email_marketing/models.py

```diff
@@ -13,6 +13,8 @@ class EmailMarketingConfiguration(ConfigurationModel):
     class Meta(object):
         app_label = "email_marketing"
 
+    CACHE_KEY = "sailthru.user.lists.data"
```

Comment:
I think you should be more specific on the name of this constant. Or be less specific for the value of this cache_key. It's in the email marketing configuration model after all. Maybe do 
`CACHE_KEY = "email_marketing_cache"`?